From 1a374fd8cfad4a312e0d000bf74bd92cfbed4fb8 Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Fri, 28 Nov 2014 14:44:38 +0800
Subject: [PATCH 5135/5965] g9tv: deselect ir tx support

not select NEW_AM_IR_TX
but maybe need to select AM_IR_IRBLASTER
---
 arch/arm/configs/mesong9tv_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/mesong9tv_defconfig b/arch/arm/configs/mesong9tv_defconfig
index 692a80cbb328..089726252739 100755
--- a/arch/arm/configs/mesong9tv_defconfig
+++ b/arch/arm/configs/mesong9tv_defconfig
@@ -151,7 +151,6 @@ CONFIG_RFKILL=y
 CONFIG_I2C_AML=y
 CONFIG_SARADC_AM=y
 CONFIG_NEW_AM_REMOTE=y
-CONFIG_NEW_AM_IR_TX=y
 CONFIG_ADC_KEYPADS_AM=y
 CONFIG_MESON_CS_DCDC_REGULATOR=y
 CONFIG_AML_PMU=y
-- 
2.19.0

